README.md

```diff
@@ -4,7 +4,11 @@ This is the product page for GovWifi. It is a [middleman site](http://middlemana
 
 ## Getting Setup for Developement
```

Comment:
can we fix the old typo in developement too? :)

Reply:
Lol its when two devs meet, fall in love and run away together....  dev-elople-ment ;)